ansible/configs/studentvm/sample_vars/studentvm_osp_bare_bones.yml

```diff
@@ -0,0 +1,32 @@
+# -------------------------------------------------------------------
+# StudentVM on OpenStack. Bare bones, only `cloud-user` will have
+# access to the VM.
+# -------------------------------------------------------------------
+
+# guid is the deployment unique identifier, it will be appended to all tags,
+# files and anything that identifies this environment from another "just like it"
+guid: defaultguid
+
+# Create an empty studentvm. The only user to have access to this
+# VM will be cloud-user
+studentvm_roles: []
+
+# Student User ID
+# Should be overwritten by CloudForms / Babylon based
+# on which user requested this configuration
+# student_name: "lab-user"
+
+# To create a student user set the following:
+# studentvm_roles:
+# - studentvm-user
+
+# Settings for role studentvm-user
+# studentvm_user_vars:
+#   # Install a user id "{{ student_name }}"
+#   user_name: "{{ student_name }}"
+#   # If no student_password is provided a randomly generated password will be used
+#   student_password: ""
+
+# Deploy in a Ansible 2.9 Virtualenv
+agnosticv_meta:
+  virtualenv: openstack-ansible-2.9
```

Comment:
Does this work? I didn't think we went back and update some of the module names that changed from 2.8 > 2.9. Like os_facts/info.

Reply:
Yes it does. And facts gets mapped into info on 2.9. It'll show a warning.